test/integration/scheduler/preemption/preemption_test.go

```diff
@@ -1017,187 +1017,6 @@ func newAlwaysFail(_ context.Context, _ runtime.Object, _ framework.Handle) (fra
 	return &alwaysFail{}, nil
 }
 
```

Comment:
Should we have a test case that to verify 
> when the scheduler finds Pods don't fit on .status.nominatedNodeName, it tries to check other nodes as well later

Reply:
We already have the one in `TestPreferNominatedNode`.

Reply:
OK